src/System.Private.ServiceModel/tests/Common/Infrastructure/src/ServiceUtilHelper.cs

```diff
@@ -199,6 +216,80 @@ public static void EnsureClientCertificateInstalled()
         ThrowIfClientCertificateInstallationError();
     }
 
+    // Tries to ensure that the server machine certificate is installed into
+    // the TrustedPeople store.  It obtains the server certificate from the service
+    // utility endpoint and either installs it or verifies that a matching
+    // one is already installed.  InvalidOperationException will be thrown
+    // if an error occurred attempting to install the certificate.  This
+    // method may be called multiple times but will attempt the installation
+    // once only.
+    public static void EnsureServerCertificateInstalled()
```

Comment:
This sounds like a good discussion item.  What are your thoughts about taking it to design discussions and making a different PR for the code that generates a different cert (if that's what we decide)?  It will have an affect on the cert gen tools and could affect machine setup automation.  We might also consider what kinds of additional positive or negative tests to add.